test/extended/networking/util.go

```diff
@@ -500,7 +500,7 @@ func InOpenShiftSDNContext(body func()) {
 }
 
 func InBareMetalIPv4ClusterContext(oc *exutil.CLI, body func()) {
-	Context("when running openshift ipv4 cluster on bare metal",
+	Context("when running openshift ipv4 cluster on bare metal [apigroup:config.openshift.io]",
```

Comment:
@ingvagabund please start a doc or something with TODOs to recheck usage.  I suspect the fanout on this one may be unfortunate because microshift is baremetal.